src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/ModuleToken.cs

```diff
@@ -24,6 +25,19 @@ public ModuleToken(EcmaModule module, mdToken token)
             Module = module;
             Token = token;
         }
+        public ModuleToken(EcmaModule module, EntityHandle entityHandle)
+        {
+            Module = module;
+            Token = (mdToken)MetadataTokens.GetToken(entityHandle);
+        }
+
+        public ModuleToken(EcmaModule module, Handle handle)
+        {
+            Module = module;
+            Token = (mdToken)MetadataTokens.GetToken(handle);
+        }
+
+        public bool IsNull => Module == null;
```

Comment:
I'm a little unsure how to use this class correctly. Is it invalid to call `Handle` if `IsNull == true`?